README.md

```diff
@@ -1,3 +1,11 @@
+## Deprecation note!
+
+**Please note this repository is deprecated - this repository is no longer actively maintained**. 
```

Comment:
How about:

**Please note: this repository is deprecated and is no longer actively maintained**